docs/3. Flows/3. Flow Features/05. Integrate Google sheets in Flows.md

```diff
@@ -75,3 +75,50 @@ Click on `OK` and use the column name like below and access the value of cell va
 `@results.sheet.message_1`
 
 ![image](https://user-images.githubusercontent.com/32592458/219550825-82831f46-a239-4600-b95e-bf0533f0565b.png)
+
+
+## Write to google sheet
+
+Please find the flow created for testing purpose. The below flow is an example of the response from the user will be added directly to the google sheet. 
```

Comment:
@AmitRChavan15 can you export the flow and add json here for NGO to import this flow directly